Assets/SEE/Game/City/SEECity.cs

```diff
@@ -281,6 +288,15 @@ protected static async UniTask LoadGraphMetricsAsync(Graph graph, string csvPath
 
             p.End();
 
+            //add xml-test-metrics
```

Comment:
If the comment is necessary at all, it should be rewritten to be more understandable.

Reply:
Removed it.